packages/nx/bin/nx.ts

```diff
@@ -99,23 +100,26 @@ function main() {
 }
 
 function resolveNx(workspace: WorkspaceTypeAndRoot | null) {
+  // root relative to node_modules/nx/bin/nx,
+  const commandRoot = join(__dirname, '../../..');
```

Comment:
```suggestion
  const workingDirectory = process.cwd();
```

Reply:
Unfortunately that does not work, since it detects the working directory of the command invocation, but we are looking for globals root, which is completely different folder.

`__dirname` gives us path to `nx.js` being invoked. From there on we need to start searching from the root for `nx` installations.